### PR TITLE
Add snack bars to saving user profile 

### PR DESCRIPTION
**Pull request by MickhaelSh**
## Description
Сhanged styles in profile, added snack bars to profile

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
|203136149-fb17c3e3-4d2f-4955-a79d-565adf4fb5f6| Знімок екрана 2022-11-22 230127 |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by Pipuss — APPROVED** [review]

**Review by Yurenko — DISMISSED** [review]

File: src/components/auth-form/auth-form-button/auth-form-button.styles.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { makeStyles } from '@material-ui/core/styles';
...
   authBtn: {
     fontFamily: theme.typography.fontFamily,
-    background: theme.palette.textColor,
+    backgroundColor: theme.palette.textColor,
     color: theme.palette.backgroundColor,
     fontSize: theme.typography.button.fontSize,
     fontWeight: theme.typography.h2.fontWeight,
     lineHeight: theme.typography.h2.lineHeight,
-    height: '50px',
+    minHeight: '42px',
```
Comment (byavdoshniak) [inline src/components/auth-form/auth-form-button/auth-form-button.styles.js:11]: Button height should be controlled by paddings and not height property

File: src/pages/profile-page/avatar/avatar.js [hunk 2]
```diff
@@ -43,9 +43,13 @@ const Avatar = ({ setUserImageUrl, userImageUrl, setUpload, setDeleteAvatar, t }
     }
   };
 
-  const handleLableClass = () => (userImageUrl ? classes.updateLabel : classes.uploadLabel);
+  const handleLableClass = useMemo(
```
Comment (byavdoshniak) [inline src/pages/profile-page/avatar/avatar.js:46]: rename

File: src/pages/profile-page/profile-page.js [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
 import { handleClassName, initialValues } from '../../utils/handle-profile-page';
 import { useAppStyles } from '../../components/app/app.styles';
+import AuthButton from '../../components/auth-form/auth-form-button/auth-form-button';
+import { SnackBarContext } from '../../context/snackbar-context';
+import SnackbarItem from '../../containers/snackbar';
 
 const ProfilePage = () => {
   const [userImageUrl, setUserImageUrl] = useState(null);
   const [upload, setUpload] = useState(null);
+  const [update, setUpdate] = useState(false);
```
Comment (byavdoshniak) [inline src/pages/profile-page/profile-page.js:22]: rename to "updated" or "shouldUpdate"

File: src/pages/profile-page/profile-page.styles.js [hunk 4]
```diff
@@ -72,55 +78,38 @@ export const useStyles = makeStyles(({ palette }) => ({
     }
   },
   button: {
-    fontSize: '1rem',
-    lineHeight: '20px',
-    width: '100%',
     gridColumn: 'span 3',
-    textTransform: 'uppercase'
+    height: '48px'
```
Comment (byavdoshniak) [inline src/pages/profile-page/profile-page.styles.js:82]: change to padding

File: src/pages/profile-page/profile-page.styles.js [hunk 5]
```diff
@@ -72,55 +78,38 @@ export const useStyles = makeStyles(({ palette }) => ({
...
   },
   userActions: {
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'flex-start',
-    width: 445,
-    '@media (max-width: 1200px)': {
+    justifySelf: 'center',
+    maxWidth: 635,
+    width: '100%',
```
Comment (byavdoshniak) [inline src/pages/profile-page/profile-page.styles.js:90]: Width of grid children should be controlled by grid if they should have fullWidth

**Review by byavdoshniak — CHANGES_REQUESTED** [review]

File: src/components/auth-form/auth-form-button/auth-form-button.styles.js (hunk 1 above)
Comment (MickhaelSh) [reply src/components/auth-form/auth-form-button/auth-form-button.styles.js]: fixed

File: src/pages/profile-page/profile-page.js (hunk 3 above)
Comment (MickhaelSh) [reply src/pages/profile-page/profile-page.js]: fixed

File: src/pages/profile-page/profile-page.styles.js (hunk 4 above)
Comment (MickhaelSh) [reply src/pages/profile-page/profile-page.styles.js]: fixed

File: src/pages/profile-page/profile-page.styles.js (hunk 5 above)
Comment (MickhaelSh) [reply src/pages/profile-page/profile-page.styles.js]: fixed

**Review by tarasSanotskyi — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by IrynaMel — APPROVED** [review]

File: src/containers/checkout/checkout-form/checkout-form.js [hunk 6]
```diff
@@ -174,8 +187,8 @@ const CheckoutForm = ({ cartItems, promoCode, certificate, handleCashPayment })
                       InputProps={
                         field.name === 'phoneNumber'
                           ? {
```
Comment (LubomyrKashuba) [inline src/containers/checkout/checkout-form/checkout-form.js:189]: use && operator

File: src/pages/profile-page/profile-page.js [hunk 7]
```diff
@@ -90,92 +114,96 @@ const ProfilePage = () => {
...
+      Object.keys(textFields).map((name) => (
+        <TextField
+          key={name}
+          type='text'
+          name={name}
+          variant={TEXT_FIELD_VARIANT.OUTLINED}
+          value={values[name]}
+          InputProps={
+            name === 'phoneNumber'
+              ? {
```
Comment (LubomyrKashuba) [inline src/pages/profile-page/profile-page.js:128]: use && operator

Comment (MickhaelSh) [reply src/pages/profile-page/profile-page.js:128]: It should have an object prop, otherwise, we will get a warning

**Review by byavdoshniak — APPROVED** [review]

**Pull request merged**